docs/pattern/advanced-search.en-US.md

```diff
@@ -0,0 +1,55 @@
+---
+Order: 6
+Title:
+  En: Advanced Search
+  En-US: Advanced Search
```

Comment:
```
---
order: 6
title: Advanced Search
---
```

```
---
order: 6
title: 高级搜索
---
```

And change `docs/pattern/advanced-search.md` to `docs/pattern/advanced-search.zh-CN.md`.